docs/configuration.md

```diff
@@ -364,6 +376,18 @@ Permitted values are `error`, `warn`, `info` and `debug`. The default is
         region: fr
         container: containername
         rootdirectory: /swift/object/name/prefix
+      oss:
```

Comment:
Maybe this belongs in another PR, but it seems like we should push these examples into the storage driver specific documentation.